mittab/libs/assign_rooms.py

```diff
@@ -0,0 +1,64 @@
+import random
+from django.db import transaction
+from mittab.apps.tab.models import RoomCheckIn, Round, TabSettings
+from mittab.libs import errors, mwmatching, tab_logic
+
+
+def add_rooms():
+    no_seeding = TabSettings.get("disable_room_seeding", 0)
```

Comment:
nit: I'd prefer the tab setting to be `enable_room_seeding` so that the code is more intuitive, e.g. `true` == `enabled` is more intuitive than `true` == `disabled`